pages/_app.tsx

```diff
@@ -36,7 +36,7 @@ const App: FC<{
   const relayCache = pageProps.__relayCache as RecordMap | undefined;
 
   useEffect(() => {
-    Mixpanel.track('Page view', { path: asPath });
+    _track('Page view', { path: asPath });
```

Comment:
this is the only case where we use `_track` directly, since `_app.tsx` is above the AnalyticsContext, and page view tracking isn't imperative to track with authentication